src/Microsoft.Health.Fhir.Api/Controllers/AadSmartOnFhirProxyController.cs

```diff
@@ -105,6 +102,7 @@ public AadSmartOnFhirProxyController(IOptions<SecurityConfiguration> securityCon
         /// <param name="state">state URL parameter.</param>
         /// <param name="aud">aud (audience) URL parameter.</param>
         [HttpGet("authorize")]
+        [TypeFilter(typeof(SmartOnFhirAuditLoggingFilterAttribute), Arguments = new object[] { AuditEventSubType.SmartOnFhirAuthorize })]
```

Comment:
any reason why we didn't follow the same pattern in FhirController to put the attribute at the class level?

Reply:
I wanted to be able to use both dependency injection and passing parameters. 

The `FhirController` filter for audits relies on the `AuditHelper` to have gathered all the attributes to get the correct action so it can match them up in the audit. This seemed a lot more heavy handed that adding the attribute with the corresponding action on the 3 methods in this controller.